OpenEdXMobile/src/main/java/org/edx/mobile/util/DateUtil.java

```diff
@@ -60,4 +61,21 @@ public static String formatCourseNotStartedDate(String date) {
             return null;
         }
     }
+
+    /**
+     * Formats a date according to 'MMMM d' format.
+     * Example output is 'February 21'.
+     *
+     * @param millis a point in time in UTC milliseconds
+     * @return a string containing the formatted date.
+     */
+    @Nullable
+    public static String formatDateWithNoYear(long millis) {
+        try {
+            return new SimpleDateFormat("MMMM d").format(millis);
+        } catch (Exception e) {
```

Comment:
because format function can throw `IllegalArgumentException` as per doc, also to make it sync with other functions.

```
    /**
     * Formats an object to produce a string. This is equivalent to
     * <blockquote>
     * {@link #format(Object, StringBuffer, FieldPosition) format}<code>(obj,
     *         new StringBuffer(), new FieldPosition(0)).toString();</code>
     * </blockquote>
     *
     * @param obj    The object to format
     * @return       Formatted string.
     * @exception IllegalArgumentException if the Format cannot format the given
     *            object
     */
    public final String format (Object obj) {
        return format(obj, new StringBuffer(), new FieldPosition(0)).toString();
    }
```

Reply:
Then we should catch the same exception, not the parent.

> also to make it sync with other functions.

This doesn't make any sense.